Added $lanify

AardeYamz opened the pull request:
Added $lanify using image placed in the meme folder, still using tim_scale

AardeYamz committed:
Added $Lanify

Added $lanify, still using 'tim_scale'

AardeYamz committed:
Fixed lanify name

coffincw commented:
Hi Yannis,

Can you pull from master to your fork so that you have the most up to date code?  And this can be merged easily

AardeYamz committed:
Merge remote-tracking branch 'upstream/master'

AardeYamz commented:
I believe that I have done that just now.

AardeYamz committed:
$Zoom & Aliases

Added $zoomarun & $zoommarius,
Updated bot commands to take in additional aliases.

AardeYamz commented:
Added additional features and aliases

coffincw commented on src/bot.py:
```diff
@@ -528,6 +531,15 @@ def get_top_10(data):
+       Args:
+        - ctx: context that the command occured use this to access the message and other attributes
+    """
+    await ify(ctx, tim_scale, 'memes/lan/lan-face.png', 'lan-face.png', args)
```
Does not work, functions and variables from other files need to be referenced. Should be:
```suggestion
    await custom_meme.ify(ctx, face_detection.lan_scale, 'memes/lan/lan-face.png', 'lan-face.png', args)
```
Also please add this scale underneath `tim_scale` in _face_detection.py_
`lan_scale = (1.55, 0.45, 0.4)`

coffincw commented on src/bot.py:
```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
     """
     await custom_meme.zoomcam(ctx, 'memes/barrington/zoombarr.png', 'zoombarr_final.png', args)
 
+@client.command(name='zoommarius', aliases=['streamermarius', 'zoom-marr', 'zoom-mar', 'zoom-marius'])
```
```suggestion
@client.command(name='zoommar', aliases=['streamermarius', 'zoommarius', 'zoom-marr', 'zoom-mar', 'zoom-marius'])
```

coffincw commented on src/bot.py:
```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
+    """
+    await custom_meme.zoomcam(ctx, 'memes/marius/zoommarius.png', 'zoommarius_final.png', args)
+
+@client.command(name='zoommarun', aliases=['streamerarun', 'zoom-arun', 'zoom-adunna', 'zoomadunna'])
```
```suggestion
@client.command(name='zoomarun', aliases=['streamerarun', 'zoom-arun', 'zoom-adunna', 'zoomadunna'])
```

coffincw commented on src/bot.py:
```diff
@@ -546,6 +558,24 @@ def get_top_10(data):
+    await custom_meme.zoomcam(ctx, 'memes/marius/zoommarius.png', 'zoommarius_final.png', args)
+
+@client.command(name='zoommarun', aliases=['streamerarun', 'zoom-arun', 'zoom-adunna', 'zoomadunna'])
+async def zoommarius(ctx, *args):
```
```suggestion
async def zoomarun(ctx, *args):
```
duplicate function name

coffincw commented on src/bot.py:
```diff
@@ -199,8 +199,11 @@
+        '*lanify [image/url]*': 'The bot uses computer vision through the OpenCV library to put lan on identified faces in the inputed image',
         '*liamify [image/url]*': 'The bot uses computer vision through the OpenCV library to put liam on identified faces in the inputed image',
         '*zoombarr [image/url]*': 'Pastes an image of barr from Zoom onto the given image in the top right corner',
+        '*zoommarius [image/url]*': 'Pastes an image of marius from Zoom onto the given image in the top right corner',
```
```suggestion
        '*zoommar [image/url]*': 'Pastes an image of marius from Zoom onto the given image in the top right corner',
```

coffincw reviewed (changes requested):
Looks good, just need to make a few adjustments.

AardeYamz committed:
Update src/bot.py

Co-Authored-By: Caleb Coffin <coffincw@gmail.com>

AardeYamz committed:
Update src/bot.py

Co-Authored-By: Caleb Coffin <coffincw@gmail.com>

AardeYamz committed:
Update src/bot.py

Co-Authored-By: Caleb Coffin <coffincw@gmail.com>

AardeYamz committed:
Update src/bot.py

Co-Authored-By: Caleb Coffin <coffincw@gmail.com>

AardeYamz committed:
Update src/bot.py

Co-Authored-By: Caleb Coffin <coffincw@gmail.com>

AardeYamz committed:
Lan_scale

Added lan_scale

AardeYamz commented:
Made suggested modifications

coffincw reviewed (approved):
Looks good!
